### PR TITLE
Added a yapf file and formated the scripts and pytest code

### PR DESCRIPTION
**Pull request by chefsale**
Summary:
Currently there was no formater in the scripts folder. I have added the yapf
configuration file and formated the python folders.

YAPF is a code formatter mainainted and developed at Google:
https://github.com/google/yapf
and I've set it up to use the codeformat they use there, but open to other options.

Test Plan:
```
yapf -i scripts/*.py
git status
```

### CODE REVIEW

File: pytest/tests/adversarial/fork_sync.py [hunk 1]
```diff
@@ -8,91 +8,90 @@
 
 sys.path.append('lib')
 
-
 from cluster import start_cluster
```
Comment (ailisp) [inline pytest/tests/adversarial/fork_sync.py:11]: Looks great! I tried autopep8 before but it incorrectly moved sys.path.append('lib') after imports, yapf looks smart enough and don't have this problem
PS: i'd like to keep indent=4, as existing file uses that.

**Comment by SkidanovAlex** [comment]
Unless there are very strong objections, I'd prefer to keep 4 spaces.

**Comment by bowenwang1996** [comment]
Agree that 4 spaces is better.

**Comment by chefsale** [comment]
4 spaces should it be, as long as its not tabs I'm happy :) Updating the diff now.

**Comment by chefsale** [comment]
@SkidanovAlex approval needed. :)

**Pull request merged**